.github/workflows/label_pr.yml

```
@@ -0,0 +1,13 @@
+name: "Pull Request Labeler"
+on:
+  pull_request_target:
+    types:
+    - opened
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v2
```

```suggestion
    - uses: actions/labeler@main
```
It seems they stopped tagging versions and many features are in ~~master~~main

---

I was hoping I don't have to live dangerously with `main` but I guess I have no choice. Thanks for pointing that out!